catalog.yml

```diff
@@ -95,3 +95,9 @@
   html_url: https://czbiohub.github.io/de-novo-orthology-paper/
 - repo_url: https://github.com/greenelab/iscb-diversity-manuscript
   html_url: https://greenelab.github.io/iscb-diversity-manuscript/
+- repo_url: https://github.com/greenelab/covid19-review
+  html_url: https://greenelab.github.io/covid19-review/
+- repo_url: https://github.com/fmaguire/mag_sim_paper
+  html_url: https://finlaymagui.re/mag_sim_paper/
```

Comment:
@fmaguire your manuscript looked mature enough that we thought it would be nice to add it to the [Manubot Catalog](https://manubot.org/catalog/). Let us know if you'd like us not to share it here until there is an official preprint or journal version.

Reply:
Sure thing, it should be on biorxiv in the next few days anyway so no issues adding it to the catalogue.

Thanks for all your work on manubot, DOI citations are great, just need to persuade more co-authors to actually write markdown!